metricbeat/docs/modules/beat.asciidoc

```diff
@@ -5,16 +5,18 @@ This file is generated! See scripts/mage/docs_collector.go
 [[metricbeat-module-beat]]
 == Beat module
 
-The Beat module contains a minimal set of metrics to enable monitoring of any Beat or other software based on libbeat across
-multiple versions. To monitor more Beat metrics, use our {stack}
-{monitor-features}.
+There are two modules that collect metrics about any Beat or other software based on libbeat:
```

Comment:
Not sure if we should talk so explicitly about this as two modules, this can create confusion. In the [docs](7) and in the code this is a single module.
The only place where this behaves as two modules is when using `metricbeat module`, but not everyone uses this approach to configure Metricbeat.
I think we should add something explaining `xpack.enabled` option, and there we should explain that this option changes the enabled metricsets, for people using these docs as reference config.

Reply:
Sounds good. In addition to what you've said, I think it might be helpful to also mention under the `xpack.enabled` option documentation that this option can be configured via a shortcut: `metricbeat modules enable <actual_module_name>-xpack`. WDYT?

Reply:
Updated in 9b48420e8.